### ISSUE TITLE
False positive for "Header blocks must not contain blank lines"

### ISSUE THREAD

**Issue by kubawerlos** [squizlabs/PHP_CodeSniffer#2917]
Version: `PHP_CodeSniffer version 3.5.4 (stable) by Squiz`

Code snippet:
```php
<?php

namespace Foo {
    class Foo
    {
    }
}

namespace Bar {
    function bar()
    {
    }
}

```

Report:
```bash
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 7 | ERROR | [x] Header blocks must not contain blank lines
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by gsherwood** [comment]
This is actually a second duplicate of #2888
